src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx

```diff
@@ -71,10 +80,19 @@ function SearchPageSearchForm() {
   const suggestions =
     data?.unifiedSearchCompletionSuggestions?.suggestions ?? [];
 
+  const show = getURLSearchParamsFromAsPath(router.asPath).get("show");
+  const showMode = getShowMode(show);
+
   return (
-    <div className={styles.searchArea}>
-      <Text variant="h1">Mitä etsit?</Text>
-      <form role="search" className={styles.form} onSubmit={handleSubmit}>
+    <div>
```

Comment:
Does this div have a purpose?

Reply:
Yup, without it the search form gets buried under koros while in list view.